lib/rules/no-object-constructor.js

```diff
@@ -53,7 +53,8 @@ module.exports = {
 
         messages: {
             preferLiteral: "The object literal notation {} is preferable.",
-            useLiteral: "Replace with '{{replacement}}'."
+            useLiteral: "Replace with '{{replacement}}'.",
+            useLiteralAfterSemicolon: "Replace with '{{replacement}}', add implicit semicolon."
```

Comment:
"add implicit semicolon" sounds confusing to me as the semicolon that's going to be added is rather explicit than implicit (I get that we're adding `;` in place of an implicit semicolon, but still sounds confusing as we're not adding an implicit semicolon).

Maybe preceding?


```suggestion
            useLiteralAfterSemicolon: "Replace with '{{replacement}}', add preceding semicolon."
```